modules/clients/variables.tf

```diff
@@ -116,3 +116,7 @@ variable "use_autoscaling_group" {
   description = "Use autoscaling group"
   default     = false
 }
+
+variable "client_arch" {
```

Comment:
should be just `arch` since we are already in the context of client

Reply:
@assafgi - we soon be supporting arm on backends as well

Reply:
@jacky-mahadab you are inside client module here

Reply:
on the root module we will still have `client_arch`

Reply:
ok

Reply:
fixed